tests/js/server/resilience/moving-shards-with-arangosearch-view-cluster.js

```diff
@@ -381,6 +417,96 @@ function MovingShardsWithViewSuite (options) {
   }
 
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief get DBServers per shard via the view api. Returns an object which
+/// which keys are shards and which values are sets of DBServers.
+////////////////////////////////////////////////////////////////////////////////
+
+  function getViewServersPerShard(database, collection) {
+    const request = require("@arangodb/request");
```

Comment:
What is the `collection` parameter usage in this function?

Reply:
That's a relict, I'll remove the parameters.